requirements.txt

```diff
@@ -1,6 +1,19 @@
 asgiref==3.7.2
+Authlib==1.2.1
+certifi==2023.7.22
+cfgv==3.4.0
 Django==4.2.7
-govuk-frontend-django
-psycopg2-binary==2.9.9
+django-structlog==6.0.0
+django_extensions==3.2.3
+govuk-frontend-django==0.4.0
+gunicorn==21.2.0
+iniconfig==2.0.0
+josepy==1.14.0
+pluggy==1.3.0
+psycopg2==2.9.9
```

Comment:
For this to install in the dev container, I needed to add `libpq-dev` to the `install.sh` file - can you add this in as part of the `apt-get -y install` command in that file? Or I can add and push, but then I dont think I will be able to approve!